nixos/modules/services/torrent/transmission.nix

```diff
@@ -349,10 +351,10 @@ in
         MemoryDenyWriteExecute = true;
         NoNewPrivileges = true;
         PrivateDevices = true;
-        PrivateMounts = true;
+        PrivateMounts = mkDefault true;
         PrivateNetwork = mkDefault false;
         PrivateTmp = true;
-        PrivateUsers = true;
+        PrivateUsers = mkDefault true;
```

Comment:
This is not required, people can use lib.mkForce to overwrite any of those values

Reply:
It makes for a more friendly user interface though

Reply:
Yes, that's my rationale here. `mkDefault` gives a hint that a user may want to change them, but that security is by default (as `openFirewall` defaults to `false` for almost every service).

Reply:
It still hides it away from the user, and they actively have to find the service file to discover this + it's not even sure it makes a difference for their setup. It's quite a small hint.